### DISCUSSION TITLE
linux下运行v2.0.0的backend, 是不是可以把多余的log去除

### DISCUSSION THREAD

**Discussion by neoyxm** [YaoFANGUK/video-subtitle-extractor#233] [General]
v1.0.0的时候，没有如下带有os/Pos的log.我查了py里没有，估计是VideoSubFinderCli里的输出。
`os: 0_00_17_080__-1                                                                                                       
Pos: 0_00_19_280__-1
Pos: 0_00_22_920__-1
Pos: 0_00_25_400__-1
Pos: 0_00_38_920__-1
Pos: 0_00_42_040__-1
Pos: 0_00_43_360__-1`

**Comment by YaoFANGUK** [comment]
@eritpchy

**Comment by eritpchy** [comment]
改了环境脚本还在测试，你可以在端侧屏蔽哦👍

**Reply by neoyxm** [reply]
好的，不急

**Comment by eritpchy** [comment]
@YaoFANGUK https://github.com/eritpchy/videosubfinder-cli/releases/tag/5.70.3-ci

**Comment by YaoFANGUK** [comment]
updated

**Reply by neoyxm** [reply]
thanks
